packages/lib/src/vis/matrix/context.tsx

```diff
@@ -23,8 +24,15 @@ export const RenderedItemsContext = createContext<
 >(undefined);
 
 function GridProvider(props: Props) {
-  const { rowCount, columnCount, cellSize, sticky, cellFormatter, children } =
-    props;
+  const {
+    rowCount,
+    columnCount,
+    cellSize,
+    sticky,
+    rowHeaderCellsWidth,
```

Comment:
It is a bit weird to have a specific parameter for the row header cell width while the height is given by `cellSize.height`

Perhaps we should also have a specific parameter for header cell height that defaults to `cellSize.height` ? Following the same behaviour as `rowHeaderCellsWidth` ?

Reply:
Hmm you mean like provide a `rowHeaderCellSize` object through context? That makes sense. I'll give it a try,.

Reply:
Actually, since we never use `cellSize` as a whole but always `cellSize.width/height`, I'm actually going to go the other way and change `cellSize` to `cellWidth`, and use the global `CELL_HEIGHT` constant where needed. What do you think?

Reply:
Yep sounds good :+1:

Reply:
Not quite happy with the result. Merging as is for now, maybe we'll think of something better at some point, especially if we ever have to make the cell height configurable (it is currently hard-coded).